daffodil-codegen-c/src/main/scala/org/apache/daffodil/codegen/c/DaffodilCCodeGenerator.scala

```diff
@@ -289,7 +289,8 @@ object DaffodilCCodeGenerator
       case g: ElementParseAndUnspecifiedLength =>
         elementParseAndUnspecifiedLengthGenerateCode(g, cgState)
       case g: ElementUnused => noop(g)
-      case g: HexBinaryLengthPrefixed => hexBinaryLengthPrefixedGenerateCode(g.e, cgState)
+      case g: HexBinaryEndOfBitLimit if g.e.isPrefixed =>
```

Comment:
Hmm, maybe my suggestion was wrong and the primitive still wants to be something like HexBInaryLengthPrefixed so that the grammar is obvious and things like code generators/etc can use the obvious grammar names? The parsers generated don't necessarily have to match the names.

Reply:
Sorry that wasn't clear. I'm suggesting that we should keep the old grammar name `HexBinaryLengthPrefixed` so this change isn't needed. This way the grammar is a more clear for things like the code generator that examine it. But it's still fine to call the parsers `BitLimitLength/MinimumLength` etc from that grammar.